README.md

```diff
@@ -562,6 +562,8 @@ OPTIONS
 
   --web-optimize                [default: false] Enable optimization (minification) of web js/css/html
 
+  --[no-]code                   [default: true] --no-code prevents action code deployment
```

Comment:
that's auto-generated no need to add it now ? right @shazron @purplecabbage ?

Reply:
yes this is auto-generated right before publish (via the prepack npm script)

Reply:
thanks for confirming